src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/cart_status.html.twig

```diff
@@ -0,0 +1,36 @@
+{#**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-{{ column.options.text_align }}">
+  {# \d+ is 1 or more digits in a string  #}
+  {% set doesFieldContainDigits = record[column.options.field] matches '/^\\d+$/' %}
```

Comment:
We use this regex at multiples locations. Could we can't create a twig function for that ?

Reply:
Nice idea

Reply:
ContainsDigitExtension ?

Reply:
@tdavidsonas88 It can be done in an another PR.

Reply:
another PR is ok for me